app/views/unattended/provisioning_templates/provision/preseed_default.erb

```diff
@@ -125,6 +125,13 @@ d-i apt-setup/local<%= repos %>/key string http://keyserver.ubuntu.com/pks/looku
 <% end -%>
 <% end -%>
 
+<% if host_param('kt_activation_keys') && @host.operatingsystem.name == 'Debian' -%>
+# If we are using Katello for content management, then do not add updates and
+# security repos during the initial installation. Prevents us from defaulting to:
+# d-i apt-setup/services-select multiselect updates,security
+d-i apt-setup/services-select multiselect
+<% end -%>
+
```

Comment:
If you write it like this, you avoid an empty line in the generated output.
```suggestion

<% end -%>
```

Reply:
Applied.